<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagPanel</class>
 <widget class="QWidget" name="TagPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="TagWidget" name="wgtTags" native="true"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="edtTag"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnTag">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>贴标签</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="TagWidget" name="wgtTagLib" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagWidget</class>
   <extends>QWidget</extends>
   <header>broom_tag.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
